kbase-extension/static/kbase/js/widgets/appWidgets2/input/textareaInput.js

```diff
@@ -1,7 +1,7 @@
 define([
     'bluebird',
-    'kb_common/html',
-    '../validators/text',
+    'common/html',
```

Comment:
I'm sure I missed a few of these. A quick search shows that there's still > 60 that use `kb_common/html` - it might be worth just doing a quickie find-and-replace PR instead to catch everything up at once.

Reply:
Agree -- it'd be nice to be done with all the duplicated `kb_common` modules once and for all!